lesson-3/main.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require_relative 'config'
+
+class Main
+  @trains = []
+  @stations = []
+  @routs = []
+  class << self
+    attr_accessor :trains, :stations, :routs, :train, :station
+    include Menu
+
+    # begin menu function call
+    def stations_list
+      print_list(stations, STATION_MESSAGE, 'name')
+    end
+
+    def trains_list
+      print_list(trains, TRAIN_MESSAGE, 'number')
+    end
+
+    def print_list(list, message, attr_name)
+      if list.length.zero?
+        puts EMPTY_LIST_MESSAGE
+      else
+        list.each { |e| puts "#{message} #{value_by_variable_attribute(e, attr_name)}" }
+      end
+    end
+
+    def create_station
+      create_obj_add_to_array(NEW_STATION_MESSAGE, Station, stations, 'name')
+    end
+
+    def create_train(class_name)
+      create_obj_add_to_array(NEW_TRAIN_MESSAGE, class_name, trains, 'number')
+    end
+
+    def create_passanger_train
+      create_train(PassengerTrain)
+    end
+
+    def create_cargo_train
+      create_train(CargoTrain)
+    end
+
+    def choose_train
+      trains_list
+      self.train = find_by_arrtubute_value(NEW_TRAIN_MESSAGE, trains, 'number')
+    end
+
+    def choose_station
+      stations_list
+      self.station = find_by_arrtubute_value(NEW_STATION_MESSAGE, stations, 'name')
+    end
+
+    def value_by_variable_attribute(object, attr_name)
+      object.instance_variable_get("@#{attr_name.to_sym}")
+    end
+
+    def hook_carriages
+      case train.type
+      when :cargo
+        train.hook_carriage(CargoCarriage.new)
+      when :passenger
+        train.hook_carriage(PassengerCarriage.new)
+      end
```

Comment:
Does `rubocop` show you warning about [`RuboCop::Cop::Style::MissingElse`](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MissingElse)

Reply:
https://travis-ci.com/OmegaLord/studybook/builds/118563472

Reply:
:confused:
Looks strange...